src/tests/dfio/SConscript

```diff
@@ -0,0 +1,31 @@
+"""Build dfio plugin"""
+import daos_build
+from prereq_tools import GitRepoRetriever
+
+def scons():
+    """Execute build"""
+
+    Import('env', 'prereqs')
+
+    dfio_libs = ['daos_common', 'daos', 'dfs', 'gurt', 'uuid', 'dl']
+
+    denv = env.Clone()
+
+    AddOption('--dfio',
+              dest='dfio',
+              action='store_true',
+              default=False,
+              help='Include dfio plugin')
+    if GetOption("dfio"):
+        denv.Append(CCFLAGS=['-rdynamic', '-z', 'nodelet', '-fPIC', \
+                             '-I/home/huakuang/daos/_build.external/fio'])
```

Comment:
?? is this just a hack?
also, what if you are building using PREBUILT_PREFIX?
as i mentioned to you before, you should add an option to point to the fio source. if it's not set, you can check in _build_external, and if it's not there, you would skip the build.

Reply:
Dir('#/_build.external/fio').abspath is what you want here but I agree with Mohamad.   You should use it only as a fallback.   There is already an FIO_SRC sticky option defined but in some situations it isn't defined.   I think you would want the option to be sticky and to default to this location if not specified but dfio is specified.